scripts/fauxalysis_from_docking.py

```diff
@@ -42,26 +233,79 @@ def main():
-            compound_path = os.path.join(args.fragalysis_dir, compound_id)
-            bound_pdb_path = os.path.join(
-                compound_path, f"{compound_id}_bound.pdb"
+    ## Get Prepped_Receptor
+    if args.prepped_path:
+        prepped_dir_list = os.listdir(args.prepped_path)
+        for complex_id, values in best_structure_dict_all.items():
+            dirname = [
+                dirname
+                for dirname in prepped_dir_list
+                if values["Structure_Source"] in dirname
+            ][0]
+            values["Prepped_Receptor"] = os.path.join(
+                args.prepped_path,
+                dirname,
+                "prepped_receptor.oedu",
             )
-            if os.path.exists(bound_pdb_path):
-                print(bound_pdb_path)
-                shutil.copy2(bound_pdb_path, output_dir_path)
+            best_structure_dict_all[complex_id] = values
+    else:
+        prepped_dir_list = os.listdir(args.input_dir)
+        print(prepped_dir_list[0])
+        for complex_id, values in best_structure_dict_all.items():
+            dirname = [
+                dirname
+                for dirname in prepped_dir_list
+                if values["Compound_ID"] in dirname
+            ][0]
+            oligomeric_state = "dimer" if values["Dimer"] == True else "monomer"
+            compound_input_path = os.path.join(
+                args.input_dir,
+                dirname,
+                oligomeric_state,
+                values["Structure_Source"],
+            )
+            values["Prepped_Receptor"] = os.path.join(
+                compound_input_path, "predocked.oedu"
+            )
+            values["Ligand_SDF"] = os.path.join(
+                compound_input_path, "docked.sdf"
+            )
+            assert os.path.exists(values["Prepped_Receptor"])
+            assert os.path.exists(values["Ligand_SDF"])
+            best_structure_dict_all[complex_id] = values
+
+    if args.overwrite:
+        best_structure_dict = best_structure_dict_all
+    else:
+        # Filter if directory already exists:
+        best_structure_dict = {}
+        for complex_id, complex_dict in best_structure_dict_all.items():
+            if not os.path.exists(f"{args.output_dir}/{complex_id}"):
+                best_structure_dict[complex_id] = complex_dict
+            else:
+                print(
+                    f"Skipping {complex_id} since output already exists at:\n"
+                    f"\t{args.output_dir}/{complex_id}"
+                )
+
+    ## Get cmpd_to_fragalysis source dict if required
+    if args.fragalysis_dir:
+        with open(args.fragalysis_yaml, "r") as f:
+            cmpd_to_frag_dict = yaml.safe_load(f)
+    else:
+        cmpd_to_frag_dict = None
+    write_fragalysis_output(
+        args.input_dir,
+        args.output_dir,
+        best_structure_dict,
+        args.fragalysis_dir,
+        cmpd_to_frag_dict,
+    )
```

Comment:
We should use the keys here for the keyword arguments just such that if things change in the signature of the function (order or names, etc.) we can still expect the same behavior.
```suggestion
    write_fragalysis_output(
        args.input_dir,
        args.output_dir,
        best_structure_dict,
        frag_dir=args.fragalysis_dir,
        cmpd_to_frag_dict=cmpd_to_frag_dict,
    )
```
Maybe we should use keywords for *all* of the arguments here (not only the optional ones, python allows this), but whatever feels more comfortable with you in that case.